.github/workflows/static-analysis.yml

```diff
@@ -43,6 +43,10 @@ jobs:
     name: Static check
     runs-on: ubuntu-latest
     steps:
+      - name: Setup Go
+        uses: actions/setup-go@v4
```

Comment:
For what is this additional step needed?
Go setup we have already on  line 54

Reply:
yes, also reverted.